src/main/java/org/zone/event/listener/PlayerListener.java

```diff
@@ -92,25 +92,25 @@ public void onPlayerRegionCreateMove(MoveEntityEvent event, @Getter("entity") Pl
             percent = 0;
         }
 
-        final float finalPercent = percent;
+        final float finalPercent = percent / 100;
 
-        BossBar bar = ZonePlugin
+        BossBar bossBar = ZonePlugin
                 .getZonesPlugin()
                 .getMemoryHolder()
                 .getZoneBuilderBossBar(player.uniqueId())
                 .orElseGet(() -> BossBar.bossBar(Component.text(""),
                         finalPercent,
                         BossBar.Color.BLUE,
                         BossBar.Overlay.PROGRESS));
-        bar.name(costPrice.getDisplayName());
-        bar.progress(percent / 100);
-        bar.color(percent < 0.0 ? BossBar.Color.RED : BossBar.Color.GREEN);
+        bossBar.name(costPrice.getDisplayName());
+        bossBar.progress(percent / 100);
```

Comment:
use ``finalPercent`` here, save a divide calculation (which are the most resource intensive basic math calculations)

Reply:
yeah